user-commands.go

```diff
@@ -760,3 +760,70 @@ func (adm *AdminClient) TemporaryAccountInfo(ctx context.Context, accessKey stri
 	}
 	return infoResp, nil
 }
+
+// User provider types
+const (
+	InternalProvider    = "internal"
```

Comment:
Would it be better to call this as `builtin` instead of `internal` (the latter sounds like it refers to something to internal to minio system's functioning/implementation)?

Reply:
True, that is what we call them for API calls as well